src/node/rpc/memberfrontend.h

```diff
@@ -240,13 +240,14 @@ namespace ccf
       tsr(network),
       rng(tls::create_entropy())
     {
-      using jerr = jsonrpc::ErrorCodes;
+      using jstderr = jsonrpc::StandardErrorCodes;
```

Comment:
Should we use these throughout CCF? I'd rather we kept things consistent throughout all frontends/apps.

Reply:
I replicated what was here originally, but I've now removed that to use the full names for consistency elsewhere. It's easier to be consistently verbose than to expose consistent abbreviations everywhere.